support-frontend/assets/pages/contributions-landing/contributionsLandingActions.js

```diff
@@ -449,9 +449,9 @@ function regularPaymentRequestFromAuthorisation(
   const { billingCountry, billingState } = getBillingCountryAndState(authorisation, state);
 
   return {
-    firstName: state.page.form.formData.firstName || '',
-    lastName: state.page.form.formData.lastName || '',
-    email: state.page.form.formData.email || '',
+    firstName: (state.page.form.formData.firstName || '').trim(),
+    lastName: (state.page.form.formData.lastName || '').trim(),
+    email: (state.page.form.formData.email || '').trim(),
```

Comment:
Is this only an issue for recurring? (single only requires `email`, but is that ever an issue?)

Reply:
I've not seen it happen for single, we should get a payment-api alarm if it does.
I trimmed email here for completeness, but it was actually the name fields that had whitespace - perhaps some devices do this when autocompleting or something